lib/get/getCache.js

```diff
@@ -18,7 +18,7 @@ function _copyCache(node, out, fromKey) {
     Object.
         keys(node).
         filter(function(k) {
-            return k[0] !== prefix && k !== "$size";
```

Comment:
@michaelbpaulson @jhusain @sdesai this check used to catch all our "internal" keys (the ones with the [prefix](https://github.com/Netflix/falcor/blob/master/lib/internal/prefix.js)), but now internal keys (like back-references) are leaking into the result of `getCache` and confusing people :(